tests/test_waypoints_to_commands.py

```diff
@@ -0,0 +1,28 @@
+"""
+Test process
+"""
+import dronekit
+
+from modules.waypoints_to_commands import waypoints_to_commands, ACCEPT_RADIUS
```

Comment:
Nit: Better to be explicit about namespace: `from modules import waypoints_to_commands` and then prefix with `waypoints_to_commands` .